### ISSUE TITLE
Localize Datepicker

### ISSUE THREAD

**Issue by iron3oxide** [vizia/vizia#352]
As of writing, both month names and day abbreviations of the Datepicker widget are hardcoded str arrays corresponding to the English values. It would be great if one could easily (or even automatically) replace them with translated values.

**Comment by geom3trik** [comment]
Thank you for pointing this out. I've opened a PR with a fix for this. It allows localizing the months/days by adding the appropriate key/value pairs to a user translation (ftl) file. For example, in French:
```ftl
January = Janvier
```
The datepicker will now use the translation if the locale is set to do so. To see this in action, add the following to the `datepicker.rs` example within the application closure (on the `datepicker-localize` branch till merged):
```rust
cx.add_translation(langid!("fr"), include_str!("../resources/translations/fr/hello.ftl"));
cx.set_language(langid!("fr"));
```
Localization will now fallback to the key if no translation could be found, so if no translation is provided the datepicker will fallback to English.

**Comment by iron3oxide** [comment]
Should be closed by #353 (haven't tested it yet, but one can always reopen)
